challenges/arrayReverse/__tests__/array-reverse.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const arrayReverse = require('../array-reverse.js');
+
+describe('Testing the array-reverse module', () => {
+  it('Should return the supplied array with its elements in reverse order', () => {
+    const reversedArray = arrayReverse([1, 2, 3, 4, 5, 6]);
+
+    expect(reversedArray).toEqual([6, 5, 4, 3, 2, 1]);
+  });
+});
```

Comment:
Great job getting this test written, I would suggest looking back at this because there is a way to do it without .push and that would be great knowledge to have!